shared/src/metabase/mbql/schema.cljc

```diff
@@ -416,6 +416,11 @@
       validate-temporal-unit
       no-binning-options-at-top-level))
 
+(def field-options-for-identification
```

Comment:
this is not used for this PR, but maybe we just keep it for document purposes?

Reply:
I'd remove it now and recreate it when we need it. Which will probably be soon given our models foreign key project